vmb/templates/account/password_reset.html

```diff
@@ -23,4 +23,38 @@ <h1>{% trans "Password Reset" %}</h1>
 
     <p>{% blocktrans %}Please contact us if you have any trouble resetting your password.{% endblocktrans %}</p>
 {% endblock %}
+{% block content %}
+<div class="col-lg-5">
```

Comment:
Please make the `html` similar to https://coderthemes.com/hyper/saas/pages-recoverpw.html